tests/Ticket/1441TestCase.php

```diff
@@ -56,8 +56,14 @@ public function testTest()
 
         $q->execute();
 
+        $params = $q->getFlattenedParams();
+
         $this->assertEqual($q->getDql(), 'UPDATE Ticket_1441_User u SET u.password = ? u.updated_at = ? WHERE u.username = ?');
-        $this->assertEqual($q->getFlattenedParams(), array('test', date('Y-m-d H:i:s', time()), 'jwage'));
+        $this->assertEqual($params[0], 'test');
+        $this->assertEqual($params[2], 'jwage');
+        // Make sure updated_at is no more than 1 second difference from current time (if we check for exact string,
```

Comment:
As the comment mentions, we'd get intermittent failures on this test when the difference here crossed a second boundary. This should solve that (unless the difference is more than one second, but that should be very rare for this).